installer/install.iss.in

```diff
@@ -22,6 +23,7 @@ LZMAUseSeparateProcess=yes
 OutputBaseFilename={#APP_NAME+'-'+APP_VERSION}
 OutputDir={#GetEnv('USERPROFILE')}
 SolidCompression=yes
+SourceDir={#GetEnv('ROOTDIR')}

```

Comment:
We need to work from an *SDK install dir* point of view because the generated files from the `pacman_list` function have that point of view.